packages/ptokens-assets-evm/src/lib/index.ts

```diff
@@ -17,20 +18,20 @@ export function offChainFormat(_amount: BigNumber.Value, _decimals: number) {
   return BigNumber(_amount).dividedBy(BigNumber(10).pow(_decimals))
 }
 
-export async function getAccount(_web3: Web3): Promise<string> {
-  if (_web3.eth.defaultAccount) return _web3.eth.defaultAccount
-  const accounts = await _web3.eth.getAccounts()
-  return accounts[0]
-}
+// export async function getAccount(_web3: Web3): Promise<string> {
```

Comment:
No way to get the address? If not, remove the function making sure it's not used elsewhere.